lib/widgets/gee_text_dropdown.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:flutter/material.dart';
+import 'package:geeruh/theme.dart';
+
+class GeeTextDropdown extends StatefulWidget {
+  const GeeTextDropdown(
+      {super.key,
+      this.width = 180,
+      this.heigth = 40,
+      required this.items,
+      required this.initialValue,
+      required this.onChanged});
+
+  final double width;
+  final double heigth;
+  final List<String> items;
+  final String initialValue;
+  final Function onChanged;
+
+  @override
+  State<GeeTextDropdown> createState() => _GeeTextDropdown();
+}
+
+class _GeeTextDropdown extends State<GeeTextDropdown> {
+  double get width => widget.width;
+  double get heigth => widget.heigth;
+  String get initialValue => widget.initialValue;
+  String currentValue = "Empty";
```

Comment:
Hardcodowanie domyślnej wartości w ten sposób to karkołomny pomysł. Proponuję tu ustawić pusty string, a w puste pole dać półprzezroczystego hinta

Reply:
Nie można tak zrobić, domyślna wartość nie może być pusta, a dodatkowo musi pochodzić ze zbioru, który przekazujemy jako itemy.
Chciałem currentValue ustawić tutaj na initialValue, ale nie mogę się dobrać do widget.initialValue, ponieważ widget nie jest jeszcze zainicjowany